commcare/src/main/java/org/motechproject/commcare/service/CommcareCaseService.java

```diff
@@ -58,12 +58,33 @@
     List<CaseInfo> getCasesByUserId(String userId, Integer pageSize, Integer pageNumber, String configName);
 
     /**
+     * Query CommCareHQ for all cases under a given user id.
+     *
+     * @param ownerId  the user id from CommCareHQ
+     * @param pageSize  the size of the page
+     * @param pageNumber  the number of the page
+     * @param configName  the name of the configuration used for connecting to CommcareHQ, null means default configuration
+     * @return  the list of CaseInfo objects representing cases under the given user id and page found on the given
+     *          CommcareHQ configuration
+     */
+
+
+    List<CaseInfo> getCasesByOwnerId(String ownerId, Integer pageSize, Integer pageNumber, String configName);
+
+    /**
      * Same as {@link #getCasesByUserId(String, Integer, Integer, String) getCasesByUserId} but uses default Commcare
      * configuration.
      */
     List<CaseInfo> getCasesByUserId(String userId, Integer pageSize, Integer pageNumber);
 
     /**
+     * Same as {@link #getCasesByOwnerId(String, Integer, Integer, String) getCasesByUserId} but uses default Commcare
+     * configuration.
+     */
+
```

Comment:
What's with all those blank lines? Method signature should be just below the javadoc in the very next line